documentation/man/features/kw-deploy.rst

```diff
@@ -31,20 +31,17 @@ executed:
 
 You can specify the deploy target via command line by using the flag
 ``--remote <remote>:<port>`` (e.g., ``--remote 172.16.254.1:22``); however, if
-you do this frequently you will probably prefer to add this information to your
-local **kworkflow.config**. See the example below::
-
-  default_deploy_target=remote
-  ssh_user=root
-  ssh_ip=172.16.254.1
-  ssh_port=22
+you plan on deploying to the same target frequently you'll probably want to use
+``kw remote`` to setup and save your target's information to your local 
+configuration (located in **remote.config**) instead of manually using 
+this flag every time.
 
 If you want to install a new kernel version in your host machine, you can use
 the flag ``--local``; you will need to use your root password.
 
 Another typical operation when deploying a new kernel to a test machine, it is
 the reboot after the update. You can explicitly say it for **kw** by adding the
-flag ``--reboot``, add this to the **kworkflow.config** with::
+flag ``--reboot``, add this to the **deploy.config** with::
 
   reboot_after_deploy=yes
```

Comment:
How about replacing this example with: `kw config deploy.reboot_after_deploy yes`

Reply:
Ended up rewriting the whole paragraph as it was very confusing, but I also added the proposed example.